src/Mvc/Mvc.Formatters.Xml/src/DelegatingEnumerator.cs

```diff
@@ -41,19 +41,19 @@ public TWrapped Current
         {
             get
             {
-                object obj = _inner.Current;
+                object obj = _inner.Current!;
```

Comment:
I've done as little as possible with the Xml formatter's implementation. But the public API seems to work out alright.